content/docs/2.0.0/iofogctl/agent-config-yaml-spec.md

```diff
@@ -36,6 +36,17 @@ spec:
   bluetoothEnabled: true
   watchdogEnabled: false
   abstractedHardwareEnabled: false
+  upstreamRouters: [default-router]
```

Comment:
is default-router a string or a reference to something? it is a string, it should be "default-router" to improve clarity

Reply:
updated